build/rbac/rbac.yaml

```diff
@@ -71,7 +71,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I don't think we need to change anything in `rbac.yaml`. This file must have gotten left around when it should have been removed when we switched to auto-generating `common.yaml`.

Reply:
I didn't understand ` file must have gotten left around when it should have been removed ` can you please elaborate 

What I see is we have the wildcard there which we need to update,
Or did you mean an auto-generator should update it on it's own, by running some build command

Reply:
@BlaineEXE So we can delete rbac.yaml? (as a separate PR)
@parth-gr rbac.yaml is just obsolete, you can revert your changes to it.

Reply:
> So we can delete rbac.yaml? (as a separate PR)

Yes.